iotile_ext_cloud/iotile/cloud/cloud.py

```diff
@@ -159,7 +158,7 @@ def get_whitelist(self, device_id):
     def check_time(self, max_slop=300):
         """ Check if current system time is consistent with iotile.cloud time"""
 
-        cloud_time = requests.get('https://iotile.cloud/api/v1/server/').json().get('now', None)
+        cloud_time = self.api.session.get('https://iotile.cloud/api/v1/server/').json().get('now', None)
```

Comment:
I have forgotten about this API, which I better port to ArchFX Cloud.
Side note: We will need a way for cloud_ext not to hard code iotile.cloud as it should also be able to talk to archfx.io.
Shouldn't this be controller by the iotile config set cloud:server setting?

Reply:
Good catch.  This is the only place it's hard-coded I think but I will verify.  There is already a configurable domain as you say, it looks like this just was a bit hacky.  It's never used in production code anymore I think but I'll fix it before merging.